projects/distributed-tracing/src/shared/dashboard/data/graphql/table/table-data-source.model.ts

```diff
@@ -19,24 +26,26 @@ export abstract class TableDataSourceModel extends GraphQlDataSourceModel<TableD
 
   public getData(): Observable<TableDataSource<TableRow, SpecificationBackedTableColumnDef>> {
     return observableOf({
-      getData: request =>
-        this.query(filters => this.buildGraphQlRequest(filters, request)).pipe(
-          map(response => this.buildTableResponse(response, request))
+      getData: (request: TableDataRequest<SpecificationBackedTableColumnDef>, mode?: TableMode) =>
+        this.query(filters => this.buildGraphQlRequest(filters, request, mode)).pipe(
+          map(response => this.buildTableResponse(response, request, mode))
         ),
-      getScope: () => this.getScope()
+      getScope: (mode?: TableMode) => this.getScope(mode)
     });
   }
 
-  public abstract getScope(): string | undefined;
+  public abstract getScope(mode?: TableMode): string | undefined;
```

Comment:
mode and scope seem completely unrelated - It feels like we're just piggy backing off an existing method.